foodnet/membership/views.py

```diff
@@ -55,6 +56,42 @@ def invite(request):
     return render(request, 'foodnet/membership/invite.html', ctx)
 
 
+@login_required
+def departments_profiles(request, department_name=None):
+    # TODO: pagination of user profiles
+    assert department_name is not None
+    department = Department.objects.get(shortname=department_name)
+    user = UserProfile.get_for_user(request.user)
+    if not user.has_admin_permission(department=department):
+        return HttpResponseForbidden(content="Not a department admin.")
+
+    all_profiles = department.profiles
+    paginator = Paginator(all_profiles, 25)  # configurable?
+
+    page = request.GET.get('page')
+    try:
+        profiles = paginator.page(page)
+    except PageNotAnInteger:
+        profiles = paginator.page(1)
+    except EmptyPage:
+        profiles = paginator.page(paginator.num_pages)
+
+    ctx = {'profiles': profiles}
+    return render(request, 'foodnet/department/profiles.html', ctx)
+
+
+@login_required
+def admin_profile(request, user_id=None):
```

Comment:
For clarification, this view is a stub. It's a different user story:
https://www.pivotaltracker.com/story/show/94087128